javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -110,6 +113,19 @@ const BasicCourseTable = ({ classes, checks, displayQuarter, allowExport }) => {
     }
   }
 
+  
+  const { getAccessTokenSilently: getToken } = useAuth0();
+  const { data: roleInfo } = useSWR(
+    ["/api/myRole", getToken],
+    fetchWithToken
+  );
+  const isAdmin = roleInfo && roleInfo.role.toLowerCase() === "admin";
+  const isMember = roleInfo && roleInfo.role.toLowerCase() === "member";
+  const isGuest = roleInfo && roleInfo.role.toLowerCase() === "guest";
+  const isLoggedIn = isAdmin || isMember || isGuest;
```

Comment:
Guests aren't able to add items to their schedule, so we need to change this to reflect that. Also, now that it's shorter, it would be nicer if we could make this one line instead of having to spread it out.